code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -225,3 +240,125 @@
 	desc = "An extremely sharp blade made out of hard light. Packs quite a punch."
 	icon_state = "lightblade"
 	item_state = "lightblade"
+
+/obj/item/melee/energy/proc/nemesis_effects(mob/living/user, mob/living/target)
+	return
+
+/obj/item/melee/energy/cleaving_saw

```

Comment:
Who has this thing?

Reply:
Yes, a weaker version as his weapon and the regular as his drop.